servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/KeepAlivePolicies.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright © 2020 Apple Inc. and the ServiceTalk project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.servicetalk.http.netty;
+
+import io.servicetalk.http.netty.H2ProtocolConfig.KeepAlivePolicy;
+
+import java.time.Duration;
+
+import static io.servicetalk.http.netty.DefaultKeepAlivePolicy.DEFAULT_ACK_TIMEOUT;
+import static io.servicetalk.http.netty.DefaultKeepAlivePolicy.DEFAULT_IDLE_DURATION;
+import static java.time.Duration.ofDays;
+import static java.util.Objects.requireNonNull;
+
+/**
+ * A factory to create {@link KeepAlivePolicy} instances.
+ */
+public final class KeepAlivePolicies {
```

Comment:
This utility class is in the `http` namespace but is specific to `h2` functionality.

We should consider the following options:
1. move `KeepAlivePolicy` outside the scope of h2 specific scope (e.g. `H2ProtocolConfig`) and make it less h2 specific. I don't think the API would have to change in a material way (e.g. methods all can stay, comments may need to be adjusted). If/when we were to use it in h1 we would need some additional input from the user on sending the "ping" request, but I think that can be done independently of the `KeepAlivePolicy` API.
OR
2. clarify that this is h2 specific (e.g. add H2 in the name).

Reply:
Good point, lemme add the `H2` prefix as this is creating a `H2ProtocolConfig.KeepAlivePolicy`